### PR TITLE
Watchtowers support

### PR DESCRIPTION
**Pull request by darosior**
Still need to figure out a neat way to pass config, and then add some plugins in a directory at the root.

### CODE REVIEW

**Comment by darosior** [comment]
I'll go without config, with hardcoded examples to not needlessly complexify argument passing.

File: policies/revault_nothing.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+"""A plugin which never returns any attempt sent to it as needing to be revaulted"""
```
Comment (danielabrozzoni) [inline policies/revault_nothing.py:2]: nit: Wording is a bit weird (it took a couple of seconds to understand what you meant) but I don't have a better one so :D

File: aquarium.py [hunk 2]
```diff
@@ -29,13 +29,16 @@
...
 REVAULTD_SRC_DIR = os.path.join(SRC_DIR, "revaultd")
 REVAULT_GUI_SRC_DIR = os.path.join(SRC_DIR, "revault-gui")
 SHELL = os.getenv("SHELL", "bash")
 COORDINATORD_VERSION = os.getenv("COORDINATORD_VERSION", "master")
 COSIGNERD_VERSION = os.getenv("COSIGNERD_VERSION", "master")
 REVAULTD_VERSION = os.getenv("REVAULTD_VERSION", "master")
-REVAULT_GUI_VERSION = os.getenv("REVAULT_GUI_VERSION", "gethistory")
+REVAULT_GUI_VERSION = os.getenv("REVAULT_GUI_VERSION", "master")
+MIRADORD_VERSION = os.getenv("MIRADORD_VERSION", "master")
+REVAULT_GUI_VERSION = os.getenv("REVAULT_GUI_VERSION", "master")
```
Comment (danielabrozzoni) [inline aquarium.py:41]: Why twice revault_gui? 🤔

File: aquarium.py [hunk 3]
```diff
@@ -85,13 +88,21 @@ def build_all_binaries(build_cosig):
 
     if build_cosig:
         logging.info(
-            f"Building cosignerd at '{COSIGNERD_VERSION}' in '{COSIGNERD_SRC_DIR}'"
+            f"Building miradord at '{COSIGNERD_VERSION}' in '{COSIGNERD_SRC_DIR}'"
```
Comment (danielabrozzoni) [inline aquarium.py:91]: Mhh no, this is building cosignerd :)

**Review by danielabrozzoni — COMMENTED** [review]
Scrolled through it quickly, still didn't test :)

**Comment by darosior** [comment]
Thanks! Amended.

File: aquarium.py [hunk 4]
```diff
@@ -102,10 +112,10 @@ def build_all_binaries(build_cosig):
         build_src(
             REVAULT_GUI_SRC_DIR,
             REVAULT_GUI_VERSION,
-            "https://github.com/edouardparis/revault-gui",
+            "https://github.com/master/revault-gui",
```
Comment (edouardparis) [inline aquarium.py:115]:
```suggestion
            "https://github.com/revault/revault-gui",
```

Comment (darosior) [reply aquarium.py:115]: Thanks i'm so far away...

**Review by edouardparis — APPROVED** [review]
ACK 176fbc21e6904d83db3a1708d11ae07426820fbd tested with -policy policies/max_value_per_day.py

**Pull request merged**